docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -64,3 +64,7 @@ ReST to markdown by sphinx-gallery. This currently only works for basic ReST
 functionality, so try to limit use of ReST directives (e.g., `.. note::`),
 internal links (e.g., `:ref:`), and other functionality that won't render well
 in markdown.
+
+
+.. note::
```

Comment:
These are rst files, not the examples themselves. The issue is about adding these notes to the examples.

Reply:
Ohh, I understand. I need to add `#` before each line if I put the note in .py, right?